core/templates/components/question-directives/modal-templates/question-editor-modal.controller.spec.ts

```diff
@@ -352,13 +370,13 @@ describe('Question Editor Modal Controller', function() {
         expect($uibModalInstance.close).not.toHaveBeenCalled();
       });
 
-    it('should dismiss modal if there is no pendent changes', function() {
+    it('should dismiss modal when there is no pending changes', function() {
```

Comment:
```suggestion
    it('should dismiss modal when there are no pending changes', function() {
```

Reply:
In all the it blocks following this too.

Reply:
I've changed it before this comment, but the Outdated is not being shown.